README.md

```diff
@@ -265,6 +265,13 @@ the previous **Leader daemon** section.
 Developing
 ===
 
+Architecture
+---
+
+Before you jump into the code, review the [Solana Architecture](https://solana-labs.github.io/solana/).
+At the time of this writing the document is quite incomplete, but links to various blogs articles and
```

Comment:
I'd strike this entire sentence, to avoid having to remember to remove it later.  Instead just mark the sections in the book that are under development as such.

Reply:
done